charts/opensrp-web/values.yaml

```diff
@@ -92,7 +92,9 @@ reactEnvironmentVariables:
   REACT_APP_OPENSRP_LOGOUT_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/logout.do"
   REACT_APP_OPENSRP_USER_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/user-details/"
   REACT_APP_OPENSRP_API_BASE_URL: "https://{{ .Values.sharedVars.opensrpServer }}/opensrp/rest/"
+  REACT_APP_FHIR_API_BASE_URL: https://{{ .Values.sharedVars.fhirServer }}/fhir
```

Comment:
Whats the default value of this `.Values.sharedVars.fhirServer`?

Reply:
should be https://fhir.labs.smartregister.org/fhir